runtime/vm/CRIUHelpers.cpp

```diff
@@ -1906,15 +1913,6 @@ criuCheckpointJVMImpl(JNIEnv *env,
 			case RESTORE_ARGS_RETURN_OK:
 				break;
 			}
-
-			if (!checkTransitionToDebugInterpreter(currentThread)) {
-				currentExceptionClass = vm->checkpointState.criuJVMRestoreExceptionClass;
-				nlsMsgFormat = j9nls_lookup_message(
-					J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE,
-					J9NLS_VM_CRIU_CHECK_TRANSITION_TO_DEBUG_INTERPRETER_FAILED,
-					NULL);
-				goto wakeJavaThreadsWithExclusiveVMAccess;
```

Comment:
Previously not `transitionToDebugInterpreter()` is an error, will it just be ignored now?

Reply:
These decisions were made based on my interpretation and understanding of the information in https://github.com/eclipse-openj9/openj9/issues/19835#issuecomment-2356353612